kivy/uix/modalview.py

```diff
@@ -70,19 +72,27 @@ def my_callback(instance):
     keyboard if the :attr:`ModalView.auto_dismiss` property is True (the
     default).
 
-'''
+.. versionchanged:: 2.1.0
+    Removed `attach_to` property. In contrary to :attr:`DropDown.attach_to`
```

Comment:
```suggestion
.. versionchanged:: 2.1.0

    Removed `attach_to` property. In contrary to :attr:`DropDown.attach_to`
```

I think you need a line break there.

Reply:
Thank you very much @matham for your review!!! Sphinx does not need a line break here (see e.g. [https://kivy.org/doc/stable/api-kivy.core.text.html](https://kivy.org/doc/stable/api-kivy.core.text.html).

Reply:
Reworded and moved it to the `ModalView` class comment, also to resolve your other change request (keep the property, and mark it as deprecated).